index.js

```diff
@@ -64,23 +59,30 @@ function guess() {
 	let error = document.getElementById("form-errors");
 
 	error.classList.add("error");
-	if (userInput.length === 0) {
-		error.innerHTML = "Input cannot be empty"
+	if (userInput.length == 0) {
+		error.innerHTML = "Input cannot be empty";
 	} else if (isNaN(userInput)) {
-		error.innerHTML = "Input must be a number"
-	} else if (userInput.length !== difficulty) {
-		error.innerHTML = `Input must be ${difficulty} characters`
+		error.innerHTML = "Input must be a number";
+	} else if (userInput.length != difficulty) {
+		console.log(userInput.length + " " + difficulty);
+		error.innerHTML = "Input must be " + difficulty + " characters";
 	} else if ([...new Set(userInput.split(""))].join("").length !== userInput.length) {
-		document.getElementById("form-errors").innerHTML = "Cannot have duplicate values";
-		return;
+		error.innerHTML = "Cannot have duplicate values";
 	} else {
 		error.classList.remove("error")
 		error.innerHTML = "";
-		if (win) {
-			return;
+		
+		let result = countDeadAndWounded(userInput);
+		
+		if (result.dead == difficulty) {
+			addResultRow(userInput, "All dead, you win!");;
 		} else {
-			addResultRow(userInput, deadAndWounded(compInput, userInput));
+			addResultRow(userInput, "dead: " + result.dead + " | wounded: " + result.wounded);
```

Comment:
If the result display text is changed, the instructions should be updated to reflect the change